README.md

```diff
@@ -48,117 +83,324 @@ or
+
+To get a full list of options available and their defaults please please refer to the help documentation of the submodule
+
+```
+ampcombi signal_peptide --help
+```
+
+Example usage: 
+```console
+ampcombi signal_peptide \
+--signalp_model path/to/signalp_model/ \
+--ampcombi_cluster path/to/Ampcombi_summary_cluster.tsv \
+--log true
+```
+The input option `--ampcombi_cluster` requires the 'Ampcombi_summary_cluster.tsv' which is generated by the `ampcombi cluster` submodule. 
+
+Output:
+
+The output will be written into your working directory, containing the following files and folders:
+
+```console
+<pwd>/
+└── Ampcombi_summary_cluster_SP.tsv
+├── Ampcombi_summary_cluster_SP_onlyclusterswithSP.tsv
+├── Ampcombi_summary_cluster_SP_onlyclusterswithSP.tsv
+├── signalp
+|   ├── output_*.png/
+|   ├── prediction_results_index.tsv
+|   ├── prediction_results.tsv
+|   ├── representative_seq.txt
+└── Ampcombi_signalpeptide.log
+```
+
+- **Ampcombi_summary_cluster_SP.tsv** contains the contents of the cluster summary plus a column with yes/no indicating the presence of a signal peptide sequence. 
+- **Ampcombi_summary_cluster_SP_onlyclusterswithSP.tsv** contains the contents of the cluster summary plus a column with yes/no indicating the presence of a signal peptide sequence. But in this case clusters were retained only if they conatianed a hit or more with a signaling peptide.
+- **signalp** contains the results from the tool signalp in `*.png` format showing the location of the predicted signaling peptide and the `prediction_results.tsv` contains a table with the location of the signaling peptide and the identity. The `prediction_results_index.tsv` contains a table that gives an index number to every hit found in `./AMPcombi_summary_ao_human_nonhuman_clusters_SP_onlyclusterswithSP.tsv`. This can be used to rename the [localcolabfold](https://github.com/YoshitakaMo/localcolabfold) results downstream in the workflow. 
+
+---
+
+## Example runs using test files:
+
+To test the function and outpur for AMPcombi, we provide test files that can be found in `test_faa`, `test_gbk` and `test_files`.
+
+Step1: Download the test directories and unzip :
+```
+wget https://github.com/Darcy220606/AMPcombi/tree/main/test_faa.tar.gz 
+wget https://github.com/Darcy220606/AMPcombi/tree/main/test_gbk.tar.gz 
+wget https://github.com/Darcy220606/AMPcombi/tree/main/test_files.tar.gz
+
+tar -xzvf test_faa.tar.gz
+tar -xzvf test_gbk.tar.gz
+tar -xzvf test_files.tar.gz
+```
+
+Step2: Parse tables from all AMP tools. This can be produced by running the AMP workflow from [FUNCSCAN](https://github.com/nf-core/funcscan). A pipeline for predicting functional genes.
```

Comment:
Well this is in case the user wants to run it standalone. Especially that some submodules wont be added in funcscan due to license issues (i.e. SignalP).